shipitscript/ship_actions.py

```diff
@@ -20,11 +22,16 @@ def mark_as_shipped(ship_it_instance_config, release_name):
     log.info('Marking the release as shipped with {} timestamp...'.format(shipped_at))
     release_api.update(release_name, status='shipped', shippedAt=shipped_at)
 
+    # quick API check to ensure fields are correctly updated on Ship-it side
+    post_update_generic_validation(release_api, release_name,
```

Comment:
Nit: In order to avoid to duplicate the comment, I'd suggest to name the function `check_release_has_values(release_api, release_name, values)`. What do you think?

Reply:
Sure, sounds good.